components/automate-ui/src/app/modules/infra-proxy/reset-admin-key/reset-admin-key.component.html

```diff
@@ -0,0 +1,30 @@
+<chef-loading-spinner *ngIf="isLoading" size="50"></chef-loading-spinner>
+<section class="reset-admin-key">
+  <ng-container>
+      <form [formGroup]="resetKeyForm">
+        <chef-form-field>
+          <label>
+            <span class="label">New Admin Key<span aria-hidden="true">*</span></span>
+            <textarea rows="27" cols="10" chefInput placeholder="-----BEGIN RSA PRIVATE KEY -----"
+            formControlName="admin_key" data-cy="update-org-admin-key"></textarea> 
+          </label>
+          <chef-error
+            *ngIf="(resetKeyForm.get('admin_key').hasError('required') || resetKeyForm.get('admin_key').hasError('pattern')) && resetKeyForm.get('admin_key').dirty">
```

Comment:
This check includes the `pattern` property but you have not yet wired that up. You should be using `Regex.patterns.NON_BLANK` as the pattern validator so that if one enters just whitespace it forces the error message.